frontend/packages/ceph-storage-plugin/src/components/object-bucket-page/object-bucket.tsx

```diff
@@ -50,7 +51,14 @@ const OBTableRow: React.FC<RowFunctionArgs<K8sResourceKind>> = ({ obj }) => {
         {_.get(obj, 'spec.storageClassName', '-')}
       </TableData>
       <TableData className={tableColumnClasses[3]}>
-        <ResourceKebab actions={menuActions} kind={kind} resource={obj} />
+        <ResourceKebab
+          actions={menuActions}
+          kind={kind}
+          resource={obj}
+          terminatingTooltip={t(
+            'ceph-storage-plugin~The corresponding ObjectBucketClaim must be deleted first.',
```

Comment:
```suggestion
            'ceph-storage-plugin~The corresponding ObjectBucketClaim must be deleted first.',
```
I am not sure how this will be triggered? If the OB is not in Terminating phase then will we still see this message? Also the message is misleading. OBC when deleted it will cascade the deletion and OB will also get deleted so we need some UX input here. Also disabling the whole kebab might not be a good option users still might want to add labels/annotations to the resources.

Reply:
> If the OB is not in Terminating phase then will we still see this message?

No, only when the corresponding OB is undergoing deletion.

> Also the message is misleading. OBC when deleted it will cascade the deletion and OB will also get deleted so we need some UX input here.

This message is only for when the OB is attempted to be deleted before its corresponding OBC, same as we are doing for PV/PVCs.

> Also disabling the whole kebab might not be a good option users still might want to add labels/annotations to the resources.

PTAL https://github.com/openshift/console/pull/11860#discussion_r997692847.

Reply:
cc @bipuladh